### DISCUSSION TITLE
Are we required to provide the prediction of all turbines separately

### DISCUSSION THREAD

**Discussion by luckyyangrun** [PaddlePaddle/PaddleSpatial#17] [BaiduKDDCup2022] [answered]
According to the dataset report, it seems that we only need to predict and evaluate the total wind power e.g., the sum of 134 turbines. However, the baseline model only predicts and evaluates the wind power of turbine No. 0  with the default arguments, which confuses me.  So, what is the target variable of this task? Are we required to provide the prediction of all turbines separately?

**Comment by zhoujingbo** [comment, marked as answer]
Hi,

Thanks a lot for your comments. In real-life applications, we only care about the whole power of the wind farm.

However, since there are an amount of unknown/missing values, we will use the power of each wind turbine to evaluate the final results. Therefore, it is better to predict the power of 134 wind turbines at the same time. (You also can predict the power of the wind farm, and then divide the predicted result by the 134 wind turbines)
 
We have updated our baseline code according to your comments. We will update the report soon w.r.t your concerns. Thanks a lot for your suggestion.

Regards
Jingbo
